rosdep/base.yaml

```diff
@@ -1509,6 +1511,10 @@ libgazebo5-dev:
   fedora: [gazebo-devel]
   gentoo: [sci-electronics/gazebo]
   ubuntu: [libgazebo5-dev]
+libgazebo7-dev:
+  arch: [gazebo]
+  gentoo: [sci-electronics/gazebo]
```

Comment:
> Is there a way to ensure it's gazebo7 though?

In Gentoo, just by using the package name in the installation command, probably no. You can run `emerge =sci-electronics/gazebo=7.1.0` but this will be broken as soon as they update the package. Fine grained version election is done in other configuration files.

In Arch the situation is the same AFAIK.